packages/frontend/src/components/common/MigrationBanner.js

```diff
@@ -0,0 +1,160 @@
+import { isAfter } from 'date-fns';
+import React, { useEffect } from 'react';
+import { Translate } from 'react-localize-redux';
+import styled from 'styled-components';
+
+import { MIGRATION_START_DATE, MIGRATION_END_DATE } from '../../config';
+import IconAlertTriangle from '../../images/IconAlertTriangle';
+import IconBell from '../../images/IconBell';
+import IconShare from '../../images/IconShare';
+import FormButton from './FormButton';
+import Container from './styled/Container.css';
+
+const StyledContainer = styled.div`
+    color: #995200;
+    background-color: #FFECD6;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    z-index: 1000;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    a {
+        color: #995200;
+        text-decoration: underline;
+        cursor: pointer;
+
+        :hover {
+            color: #995200;
+        }
+    }
+`;
+
+const ContentWrapper =  styled(Container)`
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 0;
+
+    &>*:first-child{
+        margin-right: 10px;
+    }
+
+    @media (max-width: 992px) {
+        padding: 16px;
+    }
+
+    @media (max-width: 768px) {
+        flex-direction: column;
+        align-items: flex-start;
+        padding: 16px;
+    }
+
+    & .content {
+        display: flex;
+        align-items: flex-start;
+        flex-wrap: none;
+
+        svg{
+            margin-right: 18px;
+            min-width: 24px;
+        }
+    }
+`;
+
+const CustomButton = styled(FormButton)`
+    color: #452500 !important;
+    background-color: #FFB259 !important;
+    border: none !important;
+    white-space: nowrap;
+    padding: 11px 16px;
+    margin: 0 !important;
+    height: 40px;
+
+    @media (max-width: 768px) {
+        margin-top: 16px !important;
+    }
+`;
+
+const MigrationBanner = ({ account }) => {
+    const migrationStartDate = new Date( MIGRATION_START_DATE * 1000);
+    const migrationEndDate = new Date(MIGRATION_END_DATE * 1000);
```

Comment:
I believe the default values for `MIGRATION_START_DATE` are incorrect here. We also aren't retrieving it from `process.env` so this is not currently overridable by an env variable. We need to add `MIGRATION_START_DATE` and `MIGRATION_END_DATE` to `configFromEnvironment.js`.

Also, I suggest we do this in `envParsers.js` by adding a new `parseDateFromShell` function and adding comments to that function indicating exactly what we expect from the environment variable and returning a `Date` object if it's provided and `undefined` if it's not.